var/spack/repos/builtin/packages/cgns/package.py

```diff
@@ -39,6 +39,8 @@ class Cgns(CMakePackage):
     variant('parallel',   default=False, description='Enable parallel features')
     variant('mem_debug',  default=False, description='Enable memory debugging option')
 
+    parallel = False
```

Comment:
Can you move this to around line 18 (for readability)?

Reply:
No problem, its moved!